docs/guide/saved-views.md

```diff
@@ -0,0 +1,23 @@
+title: Saved Views Guide
+description: Sematext Cloud is a modern monitoring, log management, transaction tracing, and real user monitoring system that includes over 40 monitoring integrations. It is a suite of products that combine high-quality logging experience with other monitoring and alerting devops tools helping fix IT production issues
```

Comment:
Should be about SVs, not generic.

Reply:
I looked at few pages, and seems we use the same excerpt everywhere or maybe I hit the few pages that have the same. Will check...

Reply:
Updated the description to "Sematext Cloud is a modern monitoring, log management, transaction tracing, and real user monitoring system that includes over 40 monitoring integrations. Saved Views is a feature that allows saving filters and groups for faster access later."

Reply:
We've placed split screen under "User Guide":
<img width="317" alt="image" src="https://user-images.githubusercontent.com/1279735/99548357-d9d0e780-29b8-11eb-8034-7f145d04e665.png">

I guess saved views and split screen are pretty much the same level.